opentelemetry-sdk/src/logs/log_processor.rs

```diff
@@ -261,24 +255,12 @@ impl<R: RuntimeChannel> BatchLogProcessor<R> {
                         .await;
 
                         if let Some(channel) = res_channel {
-                            if let Err(result) = channel.send(result) {
-                                global::handle_error(LogError::from(format!(
-                                    "failed to send flush result: {:?}",
-                                    result
-                                )));
+                            if let Err(send_error) = channel.send(result) {
                                 otel_error!(
-                                    name: "batch_log_processor_flush_error",
-                                    error = format!("{:?}", result),
-                                    message = "Failed to send flush result"
+                                    name: "BatchLogProcessor.Flush.SendResultError",
```

Comment:
`BatchLogProcessor.Flush.SendResultError` -- This does not look very user friendly. They need to know that Flush failed, but this is too much into implementation detail.

Reply:
BatchLogProcessor.Flush.SendResultError indicates that the Flush `may` have been successful, but there was failure to send the flush result to the channel. This is definitely useful for the developers of Otel Rust, if not much useful for the users.

Reply:
if it is not useful for end users, then make it debug level.

> was failure to send the flush result to the channel.

These are implementation detail.

Reply:
But this is an error , how can it be at debug level?

Reply:
if an error occur with flush, then user should know about it already as Flush should return `Result`. What additional value this gives to user? 
We shouldn't send error level logs (error/warn are typically enabled by the user) to user, unless it is actionable/giving-some-information which they would not have known otherwise. Flush failures can be easily communicated back to user via returned Result.

An example on the opposite side - say metric carinality limit hit and we are dropping metrics - there is no way to indicate that to user without internal log, so it is adding additional value to user.

Reply:
Have made it warn now - so we agree to have all errors not actionable by user as warning ?

Reply:
ok. Changed it to otel_debug.